cg/meta/archive/ddn_dataflow.py

```diff
@@ -41,11 +46,22 @@ class MiriaFile(FileTransferData):
     @classmethod
     def from_file_and_sample(
```

Comment:
Feel free to disregard if this change implies extending too much the scope of this PR
```suggestion
    def create_from_file_and_sample(
```